code/__DEFINES/traits.dm

```diff
@@ -278,6 +278,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_NOBLOOD "noblood"
 /// This just means that the carbon will always have functional liverless metabolism
 #define TRAIT_LIVERLESS_METABOLISM "liverless_metabolism"
+/// This carbon can't be overdosed by chems
+#define TRAIT_OVERDOSEIMMUNE "overdose_immune"
```

Comment:
A separate trait is not the way to do this- there are multiple ways to cause chems to stop processing in your system, such as liver removal.

Furthermore, androids are not the only species that doesn't process most reagents- skeletons and zombies also don't process most standard chems. A further wrinkle is that bone hurting juice, one of the few chemicals that skeletons can process, has a special overdose effect for skeletons. Adding TRAIT_OVERDOSEIMMUNE to skeletons would block this effect, but _not_ adding TRAIT_OVERDOSEIMMUNE to skeletons would give them(/let them keep) the same issues that androids currently have, from what I can tell.

Reply:
Yes, but the problem comes from the fact that the chemicals are not processed in the android’s body, which is why the overdose occurs endlessly. So I thought that to solve this problem it would be better to remove the overdose debuffs instead of making the skin impenetrable to epipenes. 

I didn’t know about the skeletons and their relationship to overdose. Do skeletons process chemicals when they enter their blood? (Are chemical units spent per tick). If you spend money, you can try to do something else.

Reply:
Skeletons do not process (most) chems, no. Neither do zombies.

Reply:
The solution here I think is that androids should be given a liver that just processes things differently

Reply:
This isn't really an android-specific issue. Any carbon that doesn't actually metabolize chems can still start overdosing on those chems. This seems like an obvious oversight. Its like if `on_mob_metabolize` was called on chems that don't actually metabolize. I think the best solution is to just remove this check: https://github.com/tgstation/tgstation/blob/cd38067a65f4c6e7c2412b0c2c95cee62a10888e/code/modules/mob/living/carbon/human/_species.dm#L1052-L1055

There's another check in a different part of the code already that actually accounts for whether or not the reagent can get metabolized.

Reply:
Any species that processes chems should have the organ objects to support it. Even if like with skeletons it seems to make no sense, they should have a magic abstracted liver if they're able to get overdosed by some things.

Basically I'm saying `TRAIT_LIVERLESS_METABOLISM` shouldn't exist